.evergreen/config.yml

```diff
@@ -197,7 +197,7 @@ buildvariants:
 
   - name: sync-api
     display_name: "Sync API"
-    patchable: false
+    #patchable: false
```

Comment:
Will revert before merge.

Reply:
Maybe we should be running this task on all PRs? The tests don't take that long and it'd be nice to catch stuff like this earlier